test/unit/components/footer/footer-test.spec.js

```diff
@@ -0,0 +1,252 @@
+import { StageComponent } from 'aurelia-testing';
+import { bootstrap } from 'aurelia-bootstrapper';
+import { Footer } from '../../../../src/components/footer/footer-ch';
+import { DataContext } from '../../../../src/services/datacontext';
+
+export class MockDataContext {
+  response  = undefined;
+
+  registerUserEmail(email) {return Promise.resolve(this.response)}
+}
+
+describe('Test - footer : ', () => {
+
+  let testTimeout;
+  let mockElement;
+  let mockDataContext;
+  let component;
+  let viewCompose;
+  let getComponent;
+  let isComponent;
+  let viewModel;
+
+  beforeEach( () => {
+    testTimeout = 3000;
+    process.env.APP_VERSION = '1.2.3';
+    viewCompose = '<footer-ch></footer-ch>';
+    mockDataContext = new MockDataContext();
+    mockElement = {};
+
+    viewModel = new Footer(mockElement, mockDataContext);
+    getComponent = (vm) => {
+      let comp = StageComponent
+      .withResources('components/footer/footer-ch')
+      .inView(viewCompose)
+      .boundTo({viewModel: vm});
+
+      return comp;
+    };
+
+    component = getComponent(viewModel);
+    isComponent = true;
+  });
+
+  afterEach(() =>{
+    delete process.env.APP_VERSION;
+    if(isComponent) {
+      component.dispose();
+    }
+  });
+
+  test('Test instance of the component', (done) => {
+    component.create(bootstrap).then( () => {
+      expect(component.bindingContext.viewModel.showVersion).toBeFalsy();
+      done();
+    }).catch( e => { console.log(e.toString()) });
+  }, testTimeout);
+
+  // test('Test displayVersion', (done) => {
```

Comment:
Do you think we should put in why tests are failing if we plan to fix them later?

Reply:
A comment was added to those unit test
** Requires investigation since the test is running ok locally by not in the CI/CD pipeline

Reply:
Done